src/modules/information/types.ts

```diff
@@ -0,0 +1,17 @@
+import { ScopePublicity } from '@prisma/client';
+
+export type UpdateInformationProps = {
+  bio?: string;
+  avatarUrl?: string;
+  coverImageUrl?: string;
+  emailPublicity?: ScopePublicity;
```

Comment:
as disscussed, `email` in `information` is contact email, it might different from email in user, so we should allow them edit it